instrument/options.go

```diff
@@ -44,6 +47,7 @@ func NewOptions() Options {
 	logger := log.NewLevelLogger(log.SimpleLogger, log.LevelInfo)
 	return &options{
 		logger:         logger,
+		zap:            zap.L(),
```

Comment:
Thought about using `zap.Production` here, but it returns an error and I didn't see any clean way to handle that error. We could change the type signature to bubble the error up but it's very unintuitive that a call to `NewOptions` would return an error, especially since in practice it shouldn't ever return an error. Alternatively, we could panic if there was an error but that has a code smell to it. So I decided to just go with `zap.L()` as the default here.

Reply:
Sounds good